ADAL/src/urlprotocol/ADURLProtocol.m

```diff
@@ -165,7 +192,15 @@ - (void)startLoading
         _context = context;
     }
     
-    AD_LOG_VERBOSE(context.correlationId, @"%@ - host: %@", @"-[ADURLProtocol startLoading]", [self.request.URL host]);
+    if ([ADAuthorityUtils isKnownHost:self.request.URL])
+    {
+        AD_LOG_VERBOSE(context.correlationId, @"%@ - host: %@", @"-[ADURLProtocol startLoading]", [self.request.URL host]);
+    }
+    else
+    {
+        AD_LOG_VERBOSE(context.correlationId, @"-[ADURLProtocol startLoading]");
+        AD_LOG_VERBOSE_PII(context.correlationId, @"%@ - host: %@", @"-[ADURLProtocol startLoading]", [self.request.URL host]);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Split this 127 characters long line (which is greater than 120 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=objc%3ALineLength)